lib/pure/collections/sets.nim

```diff
@@ -80,6 +80,8 @@ type
     ## <#initOrderedSet,int>`_ before calling other procs on it.
     data: OrderedKeyValuePairSeq[A]
     counter, first, last: int
+  SomeSet1[T] = HashSet[T] | OrderedSet[T]
+  SomeSet2[T] = HashSet[T] | OrderedSet[T]
```

Comment:
The second argument should be ``Iterable[T]`` though.

Reply:
But unfortunately still there is no such concept into the standard library. I can define it, but is it allowed to use concepts which are still an experimental feature?

Reply:
Yes, if it's tested well. They are experimental but in the worst case they will be replaced by something that can express the same.

Reply:
Done. I also tried `proc incl*[T](s: SomeSet[T], others: varargs[Iterable[T]])` to allow usage like `someSet.incl(someIterable, someOtherIterable)`, but unfortunately it seems that currently `varargs` does not work with concept arguments.

Reply:
Well ``varargs[Iterable[T]]`` would be too much for my taste anyway.